src/Maybe.js

```diff
@@ -36,6 +36,18 @@ Maybe.isNothing = function(x) {
   return x === _nothing;
 };
 
+Maybe.toArray = function(m) {
```

Comment:
I've removed `toArray` from this PR and I'll raise a new PR in a few days time to include a separate `Foldable.js` with a handful of useful functions that operate of foldable types.